docs/CHANGELOG.md

```diff
@@ -421,75 +421,255 @@ Beta release.
-[Nico Sonack]: https://github.com/herrhotzenplotz
-[Piotr Dąbrowski]: https://github.com/yhnavein
-[Shivam Mehta]: https://github.com/maniac-en
-[coc.nvim]: https://github.com/neoclide/coc.nvim
-[config-global-groups]: https://quick-lint-js.com/config/#global-groups
-[tiagovla]: https://github.com/tiagovla
-[wagner riffel]: https://github.com/wgrr
-
-[E0001]: https://quick-lint-js.com/errors/E0001/
-[E0003]: https://quick-lint-js.com/errors/E0003/
-[E0013]: https://quick-lint-js.com/errors/E0013/
-[E0016]: https://quick-lint-js.com/errors/E0016/
-[E0019]: https://quick-lint-js.com/errors/E0019/
-[E0020]: https://quick-lint-js.com/errors/E0020/
-[E0026]: https://quick-lint-js.com/errors/E0026/
-[E0036]: https://quick-lint-js.com/errors/E0036/
-[E0038]: https://quick-lint-js.com/errors/E0038/
-[E0040]: https://quick-lint-js.com/errors/E0040/
-[E0053]: https://quick-lint-js.com/errors/E0053/
-[E0054]: https://quick-lint-js.com/errors/E0054/
-[E0057]: https://quick-lint-js.com/errors/E0057/
-[E0060]: https://quick-lint-js.com/errors/E0060/
-[E0073]: https://quick-lint-js.com/errors/E0073/
-[E0094]: https://quick-lint-js.com/errors/E0094/
-[E0104]: https://quick-lint-js.com/errors/E0104/
-[E0106]: https://quick-lint-js.com/errors/E0106/
-[E0108]: https://quick-lint-js.com/errors/E0108/
-[E0110]: https://quick-lint-js.com/errors/E0110/
-[E0111]: https://quick-lint-js.com/errors/E0111/
-[E0119]: https://quick-lint-js.com/errors/E0119/
-[E0144]: https://quick-lint-js.com/errors/E0144/
-[E0151]: https://quick-lint-js.com/errors/E0151/
-[E0173]: https://quick-lint-js.com/errors/E0173/
-[E0176]: https://quick-lint-js.com/errors/E0176/
-[E0178]: https://quick-lint-js.com/errors/E0178/
-[E0179]: https://quick-lint-js.com/errors/E0179/
-[E0180]: https://quick-lint-js.com/errors/E0180/
-[E0181]: https://quick-lint-js.com/errors/E0181/
-[E0182]: https://quick-lint-js.com/errors/E0182/
-[E0183]: https://quick-lint-js.com/errors/E0183/
-[E0184]: https://quick-lint-js.com/errors/E0184/
-[E0185]: https://quick-lint-js.com/errors/E0185/
-[E0186]: https://quick-lint-js.com/errors/E0186/
-[E0187]: https://quick-lint-js.com/errors/E0187/
-[E0188]: https://quick-lint-js.com/errors/E0188/
-[E0189]: https://quick-lint-js.com/errors/E0189/
-[E0190]: https://quick-lint-js.com/errors/E0190/
-[E0191]: https://quick-lint-js.com/errors/E0191/
-[E0192]: https://quick-lint-js.com/errors/E0192/
-[E0193]: https://quick-lint-js.com/errors/E0193/
-[E0194]: https://quick-lint-js.com/errors/E0194/
-[E0195]: https://quick-lint-js.com/errors/E0195/
-[E0196]: https://quick-lint-js.com/errors/E0196/
-[E0197]: https://quick-lint-js.com/errors/E0197/
-[E0198]: https://quick-lint-js.com/errors/E0198/
-[E0199]: https://quick-lint-js.com/errors/E0199/
-[E0203]: https://quick-lint-js.com/errors/E0203/
-[E0205]: https://quick-lint-js.com/errors/E0205/
-[E0207]: https://quick-lint-js.com/errors/E0207/
+## Contributors
+
+- [AidenThing](https://github.com/AidenThing)<br>
```

Comment:
Must fix: A visible list doesn't belong in CHANGELOG.md. We already have AUTHORS.md for the contributor list. Undo this change.